runtime/lua/vim/autoread.lua

```diff
@@ -0,0 +1,165 @@
+-- Create a file watcher, each watcher is identified by the buffer number of the file it is watching.
```

Comment:
is this a public module or should it be prefixed with underscore?

Reply:
Yep, watch works too. I've named it `_watch.lua` for now. Although I think a lot of interesting things can be built on top of this, I am not so sure if it is good enough to be made public just yet.